Video removed from the homepage and Website Refactored

Computer8543 opened the pull request:
**As** someone engaging the Mindset Dojo website
**I need** the Video removed from the homepage and Website Refactored
**So that** the site experience is more cohesive

**Acceptance Criteria:**
- HTML renders on the website
- Renders properly on Chromium and WebKit browsers
    - Desktop
    - Mobile
- Video is no longer on the homepage
- The page looks more cohesive without the video.

My fork of the website link: https://computer8543.github.io/mindset-dojo.github.io/
 
Screenshot:
![Screenshot 2025-05-14 at 07-01-08 Conversational Leadership Revealed](https://github.com/user-attachments/assets/d6380228-cb52-44d2-a739-6f068b64e598)

The homepage could use further refinement, like removing the yellow div, but I wanted to get rid of the video first and refine this further in the pull request.

michael-basil commented:
@anuolu @abkaufmann @fzuill @aphor @billwestfall @jimmanzzullo 

Please find above my black belt claim.

I have pressure tested this with each of you and several people outside of the community.

One notable person is Alfred Darby.  Another is a writing coach.   

I feel that all the criteria has been met in spirit, though am happy to have a discussion at the next circle meeting (not this Friday, but after that).

Michael Basil 🌿

billwestfall reviewed (commented):
+1, comprehensive reflection. It's interesting how you are on a path wildly different than mine but you can communicate clearly where you are, like a well drawn map

billwestfall reviewed (approved):
+1

abkaufmann reviewed (approved):
Request reviewed and approved!

Computer8543 committed:
Remove video from index.md.

Computer8543 committed:
Removed CSS for YouTube player from md.css.

Computer8543 committed:
Removed YouTube player Javascript code from md.js.

Computer8543 committed:
Delete assets/video-previews directory

YouTube video was removed, so directory and content within is no longer needed.

Computer8543 committed:
Delete assets/yt.webp

YouTube video no longer in Website, so asset no longer needed.

Computer8543 committed:
Changed plain hyperlink to community page to button in 404.md.

Computer8543 committed:
Remove "Schedule an Introductory Call" button from the bottom of index.md.

Computer8543 committed:
Moved "Explore Impact" and "View Program" buttons to the bottom of index.md.

Computer8543 committed:
Changed "Explore Impact" button to "Explore Community" button.

Computer8543 committed:
Changed "See Community" button label to "Explore Community" in 404.md.

Computer8543 committed:
Changed "Schedule an Exploratory Call" button label to "Schedule Call" in bottom of impact.md.

Computer8543 committed:
Add "View Program" button to bottom of impact.md.

Computer8543 committed:
Change "Schedule an Exploratory Call" button label to "Schedule Call" at bottom of program.md.

Computer8543 committed:
Remove 1 of 2 < symbols from beginning of <a> tag for the "Explore Community" button at the bottom of index.md.

Computer8543 committed:
Add "Explore Community" button to bottom of program.md.

Computer8543 committed:
Change "Schedule an Exploratory Call" button label to "Schedule Call" in middle of community.md.

Computer8543 committed:
Add "View Program" button to middle of community.md.

Computer8543 committed:
Add .button-vertical-divider class to md.css.

Computer8543 committed:
Added Vertical Divider between buttons in index.md.

Computer8543 committed:
Copy over border value from <hr> tag into .button-vertical-divider in md.css.

Computer8543 committed:
Changed height of .button-vertical-divider from 24px to 40px to match .md-cta-group (button) class in md.css.

Computer8543 committed:
Change .button-vertical-divider class to fit better with .md-cta-group (button class) and <hr> CSS code in md.css.

Computer8543 committed:
Added mobile specific code for .button-vertical-divider that accounts for the buttons stacking vertically on mobile by making the .button-vertical-divider into an horizontal divider like the <hr> tag on md.css.

Computer8543 committed:
Add comment above .md-cta-group explaining what it does in md.css.

Computer8543 committed:
Added flex-direction attribute to .md-cta-group (button class) and set it to row so that the buttons are explicitly set up to be a row on desktop in md.css.

Computer8543 committed:
Added comment above .md-cta-group (button class) explaining what the changes do in md.css.

Computer8543 committed:
Added mobile specific code for .md-cta-group (button class) and .button-vertical-divider in md.css to account for the buttons stacking vertically.

Computer8543 committed:
Changed max-width of new media tag code for both .md-cta-group and .button-vertical-divider classes to 576px to match the old media tag code, and got rid of old redudant media tag code for .md-cta-group in md.css.

Computer8543 committed:
Remove Button Vertical Divider between the buttons at the bottom of index.md.

Computer8543 committed:
Remove references and code for button-vertical-divider in md.css.

michael-basil commented on index.md:
```diff
@@ -37,5 +33,5 @@ css_id: home
     </hgroup>
 </section>
 <div class="md-cta-group">
-    <a href="{{site.connect_url}}" target="_blank">Schedule an Exploratory Call</a>
```
For code readability (as long as it doesn't degrade the rendering), put a newline between these HTML elements:

```
<a href="./community">Explore Community</a>
<a href="./program">View Program</a> 
```

Computer8543 replied:
Implemented suggestion.

michael-basil commented on program.md:
```diff
@@ -61,5 +61,5 @@ css_id: program
 </section>
 
 <div class="md-cta-group">
```
For code readability (as long as it doesn't degrade the rendering), put a newline between these HTML elements:

```
<a href="./community">Explore Community</a>
<a href="{{site.connect_url}}" target="_blank">Schedule Call</a>
```

Computer8543 replied:
Implemented suggestion.

michael-basil commented on impact.md:
```diff
@@ -97,5 +97,5 @@ css_id: impact
 </section>
 <div class="md-cta-group">
-    <a href="{{site.connect_url}}" target="_blank">Schedule an Exploratory Call</a>
+    <a href="./program">View Program</a><a href="{{site.connect_url}}" target="_blank">Schedule Call</a>
```
For code readability (as long as it doesn't degrade the rendering), put a newline between these HTML elements:

```
<a href="./program">View Program</a>
<a href="{{site.connect_url}}" target="_blank">Schedule Call</a>
```

Computer8543 replied:
Implemented suggestion.

michael-basil commented on community.md:
```diff
@@ -10,7 +10,7 @@ css_id: community
 <p>This group is a space for leaders, professionals, and individuals committed to <strong>personal growth</strong>, <strong>relational excellence</strong>, and <strong>systems thinking</strong>. It’s designed for those who value confidentiality, connection, and thoughtful engagement as we practice navigating complexity and building trust through intentional conversation.</p>
 <div class="md-cta-group">
-    <a href="{{site.connect_url}}" target="_blank">Schedule an Exploratory Call</a>
+    <a href="./program">View Program</a><a href="{{site.connect_url}}" target="_blank">Schedule Call</a>
```
For code readability (as long as it doesn't degrade the rendering), put a newline between these HTML elements:

`<a href="./program">View Program</a><a href="{{site.connect_url}}" target="_blank">Schedule Call</a>`

Computer8543 replied:
Implemented suggestion.

michael-basil commented on 404.md:
```diff
@@ -8,4 +8,6 @@ css_id: error
-<a href="./community" class="md-cta">Explore the community</a>
\ No newline at end of file
+<div class="md-cta-group">
+    <a href="./community">Explore Community</a>
```
You make the call ... should this button go to the community page, home page or to have 2 buttons (program and community)

Computer8543 replied:
I decided that it would be best for the user to return to the home page, so that they can return to a base state to explore the rest of the site. If we try to sell the user on the program or community when they feel lost, we might lose them altogether.

Computer8543 committed:
Added <br> tag between buttons at bottom of index.md to create a line break.

Computer8543 committed:
Added <br> tag between buttons at bottom of program.md to create newline.

Computer8543 committed:
Added newline between buttons at bottom of impact.md.

Computer8543 committed:
Added newline between buttons at bottom of index.md.

Computer8543 committed:
Added newline between buttons at bottom of program.md.

Computer8543 committed:
Added newline between buttons in middle of community.md.

Computer8543 committed:
Change button to go to Homepage instead of Community page in 404.md.

Computer8543 committed:
Change text in 404.md to fit button.

michael-basil committed:
fix: Typo

michael-basil committed:
feat: Playful 404 language

michael-basil reviewed (approved):
Fantastic work @Computer8543!  This is a contribution which helps clear the brush, distill to essence and allow a cleaner user journey (path).
